model/qti/ParserFactory.php

```diff
@@ -1455,29 +1455,29 @@ private function buildFeedback(DOMElement $data, $interactive = false){
         }
 
         $attributes = $this->extractAttributes($data);
-        $myFeedback = new $feedbackClass($attributes, $this->item);
 
         if($data->nodeName == 'modalFeedback'){
             $myFeedback = new $feedbackClass($attributes, $this->item);
             $this->parseContainerStatic($data, $myFeedback->getBody());
-        } elseif ($data->nodeName == 'feedbackInline') {
+        } /*elseif ($data->nodeName == 'feedbackInline') {
             //@todo: to be implemented: <feedbackInline> {@see http://www.imsglobal.org/question/qtiv2p1/imsqti_implv2p1.html#section10008}
             common_Logger::i('Importing item that contains <feedbackInline> element.');
             $myFeedback = null;
-        } else {
+        } */else {
 
             //@todo: to be implemented: interactive feedback
+            //@todo: to be implemented: <feedbackInline> {@see http://www.imsglobal.org/question/qtiv2p1/imsqti_implv2p1.html#section10008}
             throw new UnsupportedQtiElement($data);
 
-            if($interactive){
+            /*if($interactive){
                 $container = new ContainerFeedbackInteractive();
                 $myFeedback = new $feedbackClass($container, $attributes, $this->item);
                 $this->parseContainerInteractive($data, $myFeedback->getBody());
             }else{
                 $container = new ContainerStatic();
                 $myFeedback = new $feedbackClass($container, $attributes, $this->item);
                 $this->parseContainerStatic($data, $myFeedback->getBody());
-            }
+            }*/
```

Comment:
I think we might as well as remove this comment codes. It is not pretty to release codes with this.
We want to recover this code by searching the commit history. If you want to make this simply, you may use clear key words such as "removed feedbackInline parsing" etc. what do you think of that?